ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/adapter/OnBoardingPagerAdapter.kt

```diff
@@ -0,0 +1,50 @@
+package org.cru.godtools.tutorial.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.viewpager.widget.PagerAdapter
+import org.cru.godtools.tutorial.BR
+import org.cru.godtools.tutorial.R
+import org.cru.godtools.tutorial.animation.animateToNextText
+import org.cru.godtools.tutorial.databinding.BakedInOnboardingWelcomeBinding
+import org.cru.godtools.tutorial.util.OnBoardingCallbacks
+
+class OnBoardingPagerAdapter(val callbacks: OnBoardingCallbacks) : PagerAdapter() {
```

Comment:
I'm glad you have only 1 PagerAdapter. name it something more appropriate, such as `TutorialPagerAdapter`